swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -68,6 +68,10 @@ You can publish a Ballerina archive to the <a href="https://central.ballerina.io
 
 3. Download and place the `Settings.toml` file in your home repository (`<USER_HOME>/.ballerina/`). If you already have a `Settings.toml` file configured in your home repository, follow the other option and copy the access token into the `Settings.toml`. 
```

Comment:
We can also show how the token should be placed in the file as well don't you think?
```
[central]
accesstoken="<token>"
```
So even a first time user would know how to record a token